ledger/ledger-on-sql/src/main/scala/com/daml/ledger/on/sql/migrations/V3__Backfill_Key_Hash_State_Table.scala

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.on.sql.migrations
+
+import anorm.{BatchSql, NamedParameter}
+import com.daml.ledger.on.sql.queries.StateKeyHashing
+import org.flywaydb.core.api.migration.{BaseJavaMigration, Context}
+
+import java.sql.{Connection, ResultSet}
+import scala.collection.compat.immutable.LazyList
+import scala.jdk.CollectionConverters._
+
+private[migrations] abstract class V3__Backfill_Key_Hash_State_Table extends BaseJavaMigration {
```

Comment:
I've tested this migration manually using Postgres. Do you think it's worth writing a test for this?

Reply:
Ledger-on-sql is a library used by sandbox-next only. Its postgres-backed ledger-on-sql is a configuration that is not built into any client-facing product. Long term we want to get rid of it in favour of sandbox-on-x. So writing test for it would be throwing good money after bad.